source/type/access_point.h

```diff
@@ -33,24 +33,33 @@ www.navitia.io
 #include "type/type_interfaces.h"
 #include "type/geographical_coord.h"
 
-#include <boost/container/flat_set.hpp>
-
-#include <vector>
-#include <set>
-
 namespace navitia {
 namespace type {
 
 struct AccessPoint : public Header, Nameable, hasProperties, HasMessages {
     const static Type_e type = Type_e::AccessPoint;
+
+    // parameters
+    std::string stop_code = "";
+    bool is_entrance = false;
+    bool is_exit = false;
+    uint pathway_mode = -1;
+    uint length = -1;
+    uint traversal_time = -1;
+    uint stair_count = -1;
+    uint max_slope = -1;
+    uint min_width = -1;
```

Comment:
Setting an **unsigned** int to -1 always feels strange to me.
Maybe use 0 as a default value ?

Reply:
hahaha my bad, at the start it was 0, but i needed to handle the 0, so i decided to put -1 for the default value and it is not an **uint** obviously. thanks !
Don't know why it is work here https://github.com/CanalTP/navitia/pull/3616/files#diff-65c56ad7590d85028f4c4d16f93f969c03d1640a35a333f2be0ba097ea7a2e2fR461

Reply:
It works because -1 is a 'valid' value for uint, it is equal to  UINT::Max :grimacing: